web/views.py

```diff
@@ -93,6 +170,11 @@ class DemographicDataCreateView(generic.CreateView):
     model = DemographicData
     form_class = forms.DemographicDataForm
 
+    def dispatch(self, request, *args, **kwargs):
```

Comment:
LoginRequiredMixin, this should probably end up on almost all our views that aren't the flat pages of lookit or login/signup